api-server/src/authentication/filters/remove-sensitive-data-for-non-admins.ts

```diff
@@ -1,11 +1,9 @@
 import {Request} from "express";
-import {CurrentEvent} from "../../current-events/models/current-event.model";
 import isAdminUser from "../is-admin-user";
-import {VenueModel} from "../../venues/models/venue.model";
 import EventDTO from "../../events/dto/eventDTO";
 
-type GenericEvent = EventDTO | CurrentEvent
-type GenericEventList = EventDTO[] | CurrentEvent[]
```

Comment:
I like seeing this disappear, very nice

![image](https://user-images.githubusercontent.com/1688763/162577897-9839b74e-47c1-4756-903e-913c0bf2c640.png)